packages/hash/frontend/src/pages/[account-slug]/types/entity-type/ontology-chip.tsx

```diff
@@ -98,3 +132,7 @@ export const OntologyChip = ({
     </Typography>
   </Stack>
 );
+
+const OntologyChipForwardRef = forwardRef(OntologyChip);
+
+export { OntologyChipForwardRef as OntologyChip };
```

Comment:
```suggestion
export const OntologyChip = forwardRef(RenderOntologyChip);
```

```suggestion
export const OntologyChip = forwardRef(renderOntologyChip);
```

etc.

`ForwardRefRenderFunction` is not a React component because it has two args instead of one. It's useful to call ref forwarding functions distinctly to avoid accidental misuse or confusion.

Reply:
What you're suggesting messes up the name in dev tools, hence this approach (which is already used elsewhere in the project).

We want the dev tools to look like this 

<img width="184" alt="image" src="https://user-images.githubusercontent.com/719814/195359584-e8027acd-1afb-4cf7-baaa-6247e8529b9c.png">

What you're suggesting makes it look like this

<img width="235" alt="image" src="https://user-images.githubusercontent.com/719814/195359847-b6ce2d0f-3212-4eb7-a406-f4fe7071ccac.png">

Reply:
It's also worth saying `forwardRef` is a stop gap solution. They plan to remove it. https://github.com/reactjs/rfcs/pull/107

Reply:
Oh right I did not know about that effect for the dev tools. I think I usually just call `forwardRef` inline, but I'm beginning to forget that. Need to write more react code 😅